src/Catalyst.Core.Lib.Tests/UnitTests/IO/Handlers/ProtocolMessageVerifyHandlerTests.cs

```diff
@@ -0,0 +1,88 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.KeySigner;
+using Catalyst.Core.Lib.IO.Handlers;
+using Catalyst.Core.Lib.Util;
+using Catalyst.Core.Modules.Cryptography.BulletProofs;
+using Catalyst.Protocol.Cryptography;
+using Catalyst.Protocol.IPPN;
+using Catalyst.Protocol.Wire;
+using Catalyst.TestUtils;
+using Catalyst.TestUtils.Protocol;
+using DotNetty.Transport.Channels;
+using NSubstitute;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.IO.Handlers
+{
+    public sealed class ProtocolMessageVerifyHandlerTests
+    {
+        private readonly IChannelHandlerContext _fakeContext;
+        private readonly ProtocolMessage _protocolMessageSigned;
+        private readonly IKeySigner _keySigner;
+        private readonly SigningContext _signingContext;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_signingContext' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)